docs/server/installation.rst

```diff
@@ -81,32 +28,40 @@ Setup assumptions
 
 We've made some assumptions in these instructions, adjust as needed:
 
-#. We're installing into :file:`~/dev/pootle`.  For a deployment you'd want to
-   run this from the webserver directory.
-#. We're using SQLite as its easy to setup. Though we do include instructions
-   for quickly setting up MySQL or PostgreSQL.
-#. We're setting up the key parts of Pootle including Redis, Workers and Local
-   TM.
-#. This is a test installation.  We're not setting up a server for hosting or
-   optimising in any way.
-#. We're installing using :command:`pip`. There are also
-   :ref:`instructions for setting up Pootle using a git checkout
-   <installation#git>`.
+#. All of the :ref:`system requirements <requirements#packages>` have been
+   installed.
+#. We're installing into :file:`~/dev/pootle`.
+#. We're using SQLite as its easy to setup.
```

Comment:
As I see this mistake being endlessly repeated by natives, as a recap:
* it's: contracted form of "it is" or "it has".
* its: possessive pronoun of it.